### PR TITLE
Generate notification when mentioning a user

### PR DESCRIPTION
**Pull request by karaxuna**
Change-type: minor

### CODE REVIEW

File: lib/subscriptions.ts [hunk 1]
```diff
@@ -143,66 +239,28 @@ export const evaluate = async ({
 				return;
 			}
 
-			const creatorSession = await getSession(creatorId);
-
-			if (!creatorSession) {
+			// User already got notification, because he was mentioned
+			if (getMentions(message).includes(creator.slug)) {
 				return;
 			}
```
Comment (karaxuna) [inline lib/subscriptions.ts:245]: Maybe we should just try to insert another notification and ignore duplicate contract error.

Comment (LucianBuzzo) [reply lib/subscriptions.ts:245]: I think just insert another notification, I don't think you even need to worry about using a deterministic slug. It's not a huge issue to get notified again if the contract changes I think 🤔

Comment (LucianBuzzo) [reply lib/subscriptions.ts:245]: Or just ignore the duplicate error for sure!

**Comment by LucianBuzzo** [comment]
Relies on https://github.com/product-os/jellyfish-worker/pull/1647 to be testable, as currently the mentions fields are only populated if the event was created in the UI.

**Comment by LucianBuzzo** [comment]
@balena-ci retest

**Comment by LucianBuzzo** [comment]
@balena-ci I self-certify!

**Review by LucianBuzzo — APPROVED** [review]

**Pull request merged**